addons/snailmail_account/wizard/account_invoice_send.py

```diff
@@ -11,7 +11,7 @@ class AccountInvoiceSend(models.TransientModel):
 
     partner_id = fields.Many2one('res.partner', compute='_get_partner', string='Partner')
     snailmail_is_letter = fields.Boolean('Send by Post', help='Allows to send the document by snail mail (coventional posting delivery service)', default=lambda self: self.env.user.company_id.invoice_is_snailmail)
-    snailmail_cost = fields.Float(string='Credits', compute='_snailmail_estimate', store=True, readonly=True)
+    snailmail_cost = fields.Integer(string='Stamps', compute='_snailmail_estimate', store=True, readonly=True)
     currency_id = fields.Many2one('res.currency', compute='_get_invoice_currency', string="Currency")
```

Comment:
I think this field became useless, we should get rid of it if we don't use it anymore (with the function ^^)